src/levanter/callbacks.py

```diff
@@ -23,11 +23,16 @@
 logger = logging.getLogger(__name__)
 
 
-def eval_loss_loop(loss_fn, model, dataset, max_batches: Optional[int] = None):
+def eval_loss_loop(loss_fn, model, dataset, max_batches: Optional[int] = None, name: Optional[str] = None):
     total_loss = 0.0
     n = 0
 
-    pbar = tqdm(dataset, desc="eval", position=1, leave=False)
+    if name is not None:
```

Comment:
Not sure if I capture the logic correctly, but based on [here](https://github.com/stanford-crfm/levanter/blob/1f597337c079e7960ac3ac6ba9f1b33d3c1d6030/src/levanter/data/text.py#L567), it seems that for single dataset, the eval set name will be `""` instead of None, right?

Reply:
thanks!